layers/state_tracker/cmd_buffer_state.h

```diff
@@ -216,14 +216,16 @@ class CMD_BUFFER_STATE : public REFCOUNTED_NODE {
         // VK_DYNAMIC_STATE_LINE_STIPPLE_ENABLE_EXT
         bool stippled_line_enable;
 
+        uint32_t color_write_enable_attachment_count;
```

Comment:
Initialize this in the header file. 


```suggestion
        uint32_t color_write_enable_attachment_count = 0;
```

Also does this value need to be set to 0 when reseting? I only see this value being assigned a value in 1 location.

IE should this value be set to 0 in the reset helper function.

Reply:
right below

```
 // When the Command Buffer resets, the value most things in this struct don't matter because if they are read without
 // setting the state, it will fail in ValidateDynamicStateIsSet() for us
```